documentation/slate/source/includes/apis.md

```diff
@@ -20,6 +26,48 @@ The only arguments are the ones of [paging](#paging).
 | `coverage/{lon;lat}`                    | Information about a specific region |
 
 
+<a name="datasets"></a>Datasets
+-------------------------------
+
+Very simple endpoint providing the sets of data that are used in the given coverage.
+
+Those datasets (typically from transport authority in GTFS format), each provided by a
+unique [contributor](#contributors) are forming a [coverage](#coverage).
+
+Contributor providing the dataset is also provided in the response.
+Very usefull to know all the datas that form a coverage.
+
+The only arguments are the ones of [paging](#paging).
+
+### Accesses
+
+| url | Result |
+|----------------------------------------------|-------------------------------------------|
+| `coverage/{region_id}/datasets               | List of the datasets of a specific region |
+| `coverage/{region_id}/datasets/{dataset_id}  | Information about a specific dataset      |
+
+
+<a name="contributors"></a>Contributors
+-------------------------------
+
+Very simple endpoint providing the contributors of data for the given coverage.
+
+A contributor is a data provider (typically a transport authority), and can provide multiple [datasets](#datasets).
+For example, the contributor Italian Railways will provide a dataset for the national train and some others for the regional trains.
+We will try to put them in the same [coverage](#coverage) so that we assemble them in the same journey search, using both.
+
+Very usefull to know which contributors are used in the datasets forming a coverage.
+
```

Comment:
I agree with both... Maybe it should be part of a new PR (or part of #1754 as it should allow feed_publisher in prod).
We merge this and make a new PR? @stifoon if you don't have the time to I can prepare it.